CMakeLists.txt

```diff
@@ -70,6 +70,7 @@ pkg_check_modules(UUID REQUIRED uuid>=2.34.0)
 pkg_check_modules(PROTOC REQUIRED libprotobuf-c>=1.3.0)
 pkg_check_modules(LIBSECRET REQUIRED libsecret-1>=0.20.0)
 pkg_check_modules(LIBQRENCODE REQUIRED libqrencode>=4.0.2)
+pkg_check_modules(APPINDICATOR REQUIRED ayatana-appindicator3-0.1)
```

Comment:
I'm still unsure about whether this should be mandatory or not. I think distro packagers should be able to ship otpclient without the tray icon feature (for whatever reason).